src/lightning/pytorch/utilities/imports.py

```diff
@@ -16,20 +16,20 @@
 import sys
 
 import torch
-from lightning_utilities.core.imports import compare_version, package_available, RequirementCache
+from lightning_utilities.core.imports import compare_version, module_available, RequirementCache
 
 _PYTHON_GREATER_EQUAL_3_8_0 = (sys.version_info.major, sys.version_info.minor) >= (3, 8)
 _PYTHON_GREATER_EQUAL_3_10_0 = (sys.version_info.major, sys.version_info.minor) >= (3, 10)
 # duplicated from fabric because HPU is patching it below
 _TORCH_GREATER_EQUAL_1_13 = compare_version("torch", operator.ge, "1.13.0")
 _TORCHMETRICS_GREATER_EQUAL_0_9_1 = RequirementCache("torchmetrics>=0.9.1")
 
-_HABANA_FRAMEWORK_AVAILABLE = package_available("habana_frameworks")
+_HABANA_FRAMEWORK_AVAILABLE = module_available("habana_frameworks")
 _KINETO_AVAILABLE = torch.profiler.kineto_available()
-_OMEGACONF_AVAILABLE = package_available("omegaconf")
-_POPTORCH_AVAILABLE = package_available("poptorch")
-_PSUTIL_AVAILABLE = package_available("psutil")
-_RICH_AVAILABLE = package_available("rich") and compare_version("rich", operator.ge, "10.2.2")
+_OMEGACONF_AVAILABLE = module_available("omegaconf")
+_POPTORCH_AVAILABLE = module_available("poptorch")
```

Comment:
Why update these? Isnt it just a problem with "lightning"?

Reply:
I am not sure, so I would rather switch all :otter:

Reply:
We should keep what's working. This must be an issue with `lightning`, not `package_available` as Adrian also thought